flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/data/conversion/DataStructureConverters.java

```diff
@@ -149,6 +147,10 @@
 		switch (logicalType.getTypeRoot()) {
 			case ARRAY:
 				return ArrayObjectArrayConverter.create(dataType);
+			case MULTISET:
```

Comment:
Why were these moved?

Reply:
According to the definition of `MapType` and `MultisetType`. We also allow subclasses of `Map` such as `HashMap` or `LinkedHashMap` as input parameters.
```
putConverter(LogicalTypeRoot.MAP, Map.class, MapMapConverter::createForMapType);
```
... was too strict because it only matches if the class is `Map.class`.

Reply:
Ok, understood!